conmon/conmon.c

```diff
@@ -35,6 +37,10 @@
 #include "cmsg.h"
 #include "config.h"
 
+#ifndef CGROUP2_SUPER_MAGIC
+# define CGROUP2_SUPER_MAGIC 0x63677270
```

Comment:
Space between `#` and `define`?

Reply:
I usually use it to indent C preprocessor macros (as done by https://www.gnu.org/software/cppi/)

Reply:
well clang doesn't like it, so I had to drop it :-)